src/components/sidebar/CallTreeSidebar.js

```diff
@@ -44,20 +44,27 @@ class CallTreeSidebar extends React.PureComponent<StateProps> {
     }
 
     const data = tree.getDisplayData(selectedNodeIndex);
+    // `data.selfTime` is a string, containing either a number or, if the value
+    // is 0, is '—'. So we we use isNaN on purpose (instead of Number.isNaN), to
+    // force a conversion and decide whether we should add the unit or keep the
+    // character '—'.
+    // We don't compare against '—' to avoid hardcoded values. In the future we
+    // should have a dedicated method in `tree` to recover the values we need in
+    // the format we need.
+    const selfTime = isNaN(data.selfTime)
+      ? data.selfTime
+      : data.selfTime + 'ms';
     return (
-      <div className="sidebar sidebar-calltree">
-        <h2>
-          <div className="sidebar-title">{getFunctionName(data.name)}</div>
-          <div className="sidebar-subtitle">{data.lib}</div>
-        </h2>
+      <aside className="sidebar sidebar-calltree">
+        <hgroup className="sidebar-titlegroup">
```

Comment:
I assume this `hgroup` and multiple header tags is to give semantic meaning to the sidebar. I was surprised by the tag, as I'd not seen the tag before. I'm not sure it's the proper way to mark this up. I'll leave it up to you though.

>  HTML does not have a dedicated mechanism for marking up subheadings, alternative titles or taglines. Here are the suggested alternatives. h1–h6 elements must not be used to markup subheadings, subtitles, alternative titles and taglines unless intended to be the heading for a new section or subsection. 

Source: https://www.w3.org/TR/html52/common-idioms-without-dedicated-elements.html#common-idioms-without-dedicated-elements

> The `<hgroup>` element has been removed from the HTML5 (W3C) specification, but it still is in the WHATWG version of HTML. It is partially implemented in most browsers, though, so is unlikely to go away.
However, given that a key purpose of the `<hgroup>` element is to affect how headings are displayed by the outline algorithm defined in the HTML specification—but the HTML outline algorithm is not implemented in any browsers—then the `<hgroup>` semantics are in practice only theoretical.
So the HTML5 (W3C) specification provides advice on how to mark up Subheadings, subtitles, alternative titles and taglines without using `<hgroup>`.

Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hgroup